homework3/jenkins.postman_collection.json

```diff
@@ -0,0 +1,309 @@
+{
+	"info": {
+		"_postman_id": "bf4c07ae-39a5-45cb-8ab3-45fc87507246",
+		"name": "jenkins",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "get suite",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "7edad2c6-18fb-4e98-be5b-72cf525c1ca5",
+						"exec": [
+							"const Body = pm.response.json();\r",
+							"const suiteName = `${pm.collectionVariables.get(\"name\")} - ${pm.collectionVariables.get(\"JiraProjectName\")}`;\r",
+							"\r",
+							"const TempItem = _.find(Body, function (item) {\r",
```

Comment:
You could use arrow function here instead of anonymous one;
`_.find(body, item => item.name === suiteName`